WalletWasabi/Backend/Models/Requests/InputsRequest.cs

```diff
@@ -10,11 +10,12 @@ namespace WalletWasabi.Backend.Models.Requests
 {
 	public class InputsRequest
 	{
-		[Required]
+		[Required, MinLength(1)]
 		public IEnumerable<InputProofModel> Inputs { get; set; }
 
-		[Required]
-		public IEnumerable<string> BlindedOutputScripts { get; set; }
+		[Required, MinLength(1)]
+		[JsonProperty (ItemConverterType = typeof(Uint256JsonConverter))]
```

Comment:
I spent hours trying to figure out how the hell can I do this. Finally gave up and defaulted back to strings.

Reply:
Also can we make this universal? There are other responses you missed. For example the FilterModel's

Reply:
Elegant. Meanwhile filters done by Lucas.

Reply:
Done for `FilterModel`.